docs/content/install.md

```diff
@@ -10,6 +10,7 @@ We have a few release types available for you to use:
 
 * [Latest](#latest)
 * [Canary](#canary)
+* [v1 Prerelease](#v1-prerelease)
```

Comment:
nit: what if this was just prerelease, and then within the "Prelease link it talked about v1 specifically?

Reply:
I like it, thanks!